dvc/logger.py

```diff
@@ -170,15 +175,25 @@ def setup(level=logging.INFO):
     logging.config.dictConfig(
         {
             "version": 1,
-            "filters": {"exclude_errors": {"()": ExcludeErrorsFilter}},
+            "filters": {
```

Comment:
Could you explain what is going on here in general? The commit message doesn't have any explanation.

Reply:
this was needed as previously only `logging.DEBUG` and `logging.WARNING` were handled by `dvc.logger.LoggerHandler`. However this change ensures that `logging.INFO` is also handled by our class. The custom class is required to ensure that `logger.info()` interleaves properly with `dvc.progress.Tqdm()`.

I should've described this commit as "progress-aware `logger.info`"